src/java.base/share/classes/sun/security/jca/ProviderList.java

```diff
@@ -373,13 +373,15 @@ public Service getService(String type, String name) {
                 (pList = preferredPropList.getAll(type, name)) != null) {
             for (i = 0; i < pList.size(); i++) {
                 Provider p = getProvider(pList.get(i).provider);
+                if (p == null) {
+                    continue;
+                }
```

Comment:
Not related to this cod change, but the `getAll` line on line 373 will never return null.